ur5e:
 # Publish all joint states ----------------------------------- 
  joint_state_controller: 
    type: joint_state_controller/JointStateController 
    publish_rate: 50   
   
  # Position Controllers --------------------------------------- 
  joint1_position_controller: 
    type: position_controllers/JointPositionController 
    joint: shoulder_pan
    pid: {p: 100.0, i: 0.01, d: 10.0} 
  joint2_position_controller: 
    type: position_controllers/JointPositionController 
    joint: shoulder_lift
    pid: {p: 100.0, i: 0.01, d: 10.0} 
  joint3_position_controller: 
    type: position_controllers/JointPositionController 
    joint: elbow_joint 
    pid: {p: 100.0, i: 0.01, d: 10.0} 
  joint4_position_controller: 
    type: position_controllers/JointPositionController 
    joint: wrist1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0} 
  joint5_position_controller: 
    type: position_controllers/JointPositionController 
    joint: wrist2_joint 
    pid: {p: 100.0, i: 0.01, d: 10.0} 
  joint6_position_controller: 
    type: position_controllers/JointPositionController 
    joint: wrist3_joint 
    pid: {p: 100.0, i: 0.01, d: 10.0}
